### ISSUE TITLE
Javascript bundle sometimes fails to load

### ISSUE THREAD

**Issue by sedubois** [greglobinski/gatsby-starter-hero-blog#29]
I first noticed it because of the menu which would sometimes not collapse automatically when screen is too narrow (this is only done by Javascript):

<img width="1208" alt="screen shot 2018-09-10 at 10 43 09" src="https://user-images.githubusercontent.com/4217871/45283464-a051e700-b4e6-11e8-9493-aee574044d3c.png">
 
The issue is "fixed" by reloading the page. It looks like this is the same issue as [this one](https://github.com/gatsbyjs/gatsby/issues/2954), no idea how to fix it.

**Comment by greglobinski** [comment]
@sedubois ,
I have never seen this. How often does it happen? Could you give me the link to your website? Does it persist when the 'offline' plugin is turned off?

**Comment by sedubois** [comment]
@greglobinski this is with [your website](https://gatsby-starter-hero-blog.greglobinski.com/) (see address bar in screenshot). Happened a few times. Using latest Firefox on mac.

**Comment by sedubois** [comment]
Hard to say anything more as it's erratic...

**Comment by sedubois** [comment]
Yeah, there's maybe not much to do about this one. Maybe try the steps described in https://github.com/gatsbyjs/gatsby/issues/2954. Anyway I just wanted to make it known, feel free to close 👍
